src/components/album/AlbumChat.js

```diff
@@ -1,10 +1,12 @@
 import AlbumPreview from './AlbumPreview';
 import AlbumChatBox from './AlbumChatBox';
+import { useParams } from 'react-router-dom';
 
-const AlbumChat = ({ spotifyUrl }) => (
-  <AlbumPreview spotifyUrl={spotifyUrl} wide>
-    <AlbumChatBox spotifyUrl={spotifyUrl} />
+const AlbumChat = () => {
+  const { albumId } = useParams();
+  <AlbumPreview albumId={albumId} wide>
```

Comment:
Keep in mind that `<AlbumPreview>` component is expecting the spotify URL in order to be able to get the spotify API id that it needs to render that Spotify iframe player for playing tracks from the album. Here you've changed it to pass in the album id, but since you haven't changed the actual `AlbumPreview.js` file, I expect that this might break some things.

Reply:
Same with the change below to `AlbumChatBox`. Maybe you do just want to use the album's Mongo id in the chat component, but we should make sure the props we're passing in match the props the component is expecting.